### DISCUSSION TITLE
Is "Unofficial TWRP" trustworthy/legit?

### DISCUSSION THREAD

**Discussion by RaptaG** [openandroidinstaller-dev/openandroidinstaller#336] [Polls]
My device (Xiaomi Redmi Note 9 Pro) is not supported by TWRP officially and I found a website called https://unofficialtwrp.com which offers a TWRP version [suitable for my device](https://unofficialtwrp.com/unofficial-twrp-3-7-0-for-redmi-note-9-pro).

Leave more feedback in the comments please!

**Comment by tsterbak** [comment]
I would say it's pretty trustworthy. As trustworthy software from an basically anonymous source on the internet can be, I guess. If you have strong security concerns you shouldn't use it, or even TWRP at all. But I regularly use unofficial TWRP versions build by random persons on the xda-forum and it was fine so far.

What do others think?

**Reply by RaptaG** [reply]
Thanks for the feedback, [I reposted this on Mastodon](https://kafeneio.social/@raptag/111306849112185166) too, to reach out to more people

**Comment by RaptaG** [comment]
I'll use it, since I have no other option
